src/store/modules/order/actions.ts

```diff
@@ -874,62 +874,38 @@ const actions: ActionTree<OrderState , RootState> ={
     let resp: any
 
     const params = {
-      inputFields: {
-        statusId: "SHIPMENT_SHIPPED",
-        shipmentMethodTypeId: "SHIP_TO_STORE",
-        orderFacilityId: getCurrentFacilityId()
-      },
-      viewSize: payload.viewSize ? payload.viewSize : process.env.VUE_APP_VIEW_SIZE,
-      viewIndex: payload.viewIndex ? payload.viewIndex : 0,
-      entityName: 'ShipmentAndOrderHeader',
-      noConditionFind: "Y",
-      distinct: "Y",
-      fieldList: ['shipmentId', 'firstName', 'createdDate', 'lastName', 'orderName', 'orderId']
+      keyword: payload.queryString || '',
```

Comment:
Only add keyword to the params only when searching is done, otherwise do not add it as empty string.